doc/renderManagerConfig.md

```diff
@@ -73,6 +73,10 @@ This section describes the operation of time warp, which re-projects textures fr
 
 * enabled: True to enable time warp, false to disable it.
 
+* justInTime: If time warp is enabled, true to enable just-in-time warp and false to disable it.  This uses shearing and anisotropic scaling to adjust each scan line so that it appears at the correct location, offsetting the impact of head rotation around the X and Y rotations.  This handles the fact that on some displays scan lines do not all appear at the same time but rather from one edge of the display to another.  This should be turned off for displays that flash all of their pixels at the same time.
+
+* justInTimeRotation: Specifies the edge of the display (as seen by the viewer) where the scanlines begin.  For the OSVR HDK 2.0, this is the bottom of the display (rotation 180).  For the Oculus DK2, this is the right side of the display (rotation 90).  Displays that scan out from the top will have rotation 0 and displays that scan out from the left will have rotation -90.  This does not have to do with how the pixels are sent to the display (which can be set in the operating system) but rather with how the display natively scans out.
+
```

Comment:
These really ought to be properties of the display. The presence of much besides overfill/oversample and direct mode enable/disable in RM config is basically technical debt.